getting-started/setup-building.rst

```diff
@@ -81,13 +81,24 @@ You will only need to execute these steps once:
       $ cd cpython
       $ git remote add upstream git@github.com:python/cpython.git
```

Comment:
For public repos like this, I always set the pull request URL to the HTTPS one (`https://github.com/python/cpython.git`) since it doesn't require authentication and may be slightly less performant. It does normally require an extra step to add the URL, but in this case since the `push` URL is being overridden anyway, I figure we may as well just use it here instead, which will add an additional protection against accidentally pushing to `main` in case users don't do the next step.

```suggestion
      $ git remote add upstream https://github.com/python/cpython.git
```

Reply:
I'm a little wary of adding another bit of variance here. If we change it here, it needs to change in other steps and the copy of these steps in the bootcamp. Also, changing it here implies that we could (should?) also change it to https for fetch for `origin` as well (for symmetry). I'll take the minimum step from what you're proposing.